### PR TITLE
Story/dfc 4100 timeout restructure

### PR DESCRIPTION
**Pull request by Gurnaik-Randhawa**
Restructured to do complete job profiles in batches.

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Data/Interfaces/IJobProfileSocCodeRepository.cs [hunk 1]
```diff
@@ -6,12 +6,14 @@ namespace DFC.Digital.Data.Interfaces
 {
     public interface IJobProfileSocCodeRepository
     {
-        IQueryable<ApprenticeVacancy> GetBySocCode(string socCode);
+        IQueryable<ApprenticeVacancy> GetApprenticeVacanciesBySocCode(string socCode);
 
         RepoActionResult UpdateSocOccupationalCode(SocCode socCode);
 
-        IQueryable<SocCode> GetLiveSocCodes();
+        IQueryable<SocCode> GetSocCodes();
```
[the remark this answers is not in the document]
Comment (Gurnaik-Randhawa) [reply DFC.Digital/DFC.Digital.Data/Interfaces/IJobProfileSocCodeRepository.cs]: Done - the GetLiveSocCodes has been removed

File: DFC.Digital/DFC.Digital.Repository.ONET/DataModel/OnetSkillsFramework.Context.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@ public partial class OnetSkillsFramework : DbContext
         public OnetSkillsFramework()
             : base("name=DFC.Digital.OnetSkillsFramework")
         {
+            Database.CommandTimeout = int.MaxValue;
```
[the remark this answers is not in the document]
Comment (Gurnaik-Randhawa) [reply DFC.Digital/DFC.Digital.Repository.ONET/DataModel/OnetSkillsFramework.Context.cs]: Set to 180 seconds

File: DFC.Digital/DFC.Digital.Data/Interfaces/IJobProfileSocCodeRepository.cs [hunk 3]
```diff
@@ -6,12 +6,14 @@ namespace DFC.Digital.Data.Interfaces
 {
     public interface IJobProfileSocCodeRepository
     {
-        IQueryable<ApprenticeVacancy> GetBySocCode(string socCode);
+        IQueryable<ApprenticeVacancy> GetApprenticeVacanciesBySocCode(string socCode);
 
         RepoActionResult UpdateSocOccupationalCode(SocCode socCode);
```
[the remark this answers is not in the document]
Comment (Gurnaik-Randhawa) [reply DFC.Digital/DFC.Digital.Data/Interfaces/IJobProfileSocCodeRepository.cs]: UpdateSocOccupationalCode(SocCode socCode) is still used. 
ISocMappingRepository  is just for the SQL DB table methods - this is required for Sitefinity

[the remark this answers is not in the document]
Comment (trevorkapswarah) [reply DFC.Digital/DFC.Digital.Data/Interfaces/IJobProfileSocCodeRepository.cs]: Ready for release

**Pull request merged**